modules/sdk/project-templates/project-templates-portletmvc4spring-thymeleaf-portlet/src/main/resources/archetype-resources/src/main/java/dto/User.java

```diff
@@ -0,0 +1,35 @@
+package ${package}.dto;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+public class User implements Serializable {
```

Comment:
I know you asked me about this... but can we go ahead and format this BCHan style?  I feel it is the best way to keep everything consistent.  Is that doable?  If you don't have the cycles i can try to see if someone else could help out.

Reply:
@gamerson Regarding BChan style, do you simply mean private stuff at the bottom?